[docs] Update UI Next steps page (#30641)
expo/expo: docs/pages/develop/user-interface/next-steps.mdx

/>

<BoxLink
  title="Use TypeScript"
  Icon={Settings02Icon}

>   description="An in-depth guide on configuring an Expo project with TypeScript and includes instructions to migrate an existing JavaScript project."

Reviewer: ```suggestion
  description="An in-depth guide on configuring an Expo project with TypeScript or migrating an existing JavaScript project."
```

Wording tweak suggestion.

Author: Good catch!

How about "Learn about configuring an Expo project or migrating an existing JavaScript project to TypeScript."?

It's not an in-depth guide for learning purposes since it mostly contains migration steps.
